addon/templates/components/frost-list.hbs

```diff
@@ -14,11 +14,11 @@
       }}
     {{/if}}
 
-    {{#if (and pagination itemExpansion)}}
+    {{#if (and pagination (or itemExpansion isAnyTypedItemExpansion))}}
```

Comment:
Ahh ok i understand. Can we manage itemExpansion as `one` of the `itemExpansions`? What I mean by this is that we should simply do `isAnyItemExpansion` and hide the type thing vs non type thing in the CP.

Reply:
I just addressed this by moving the logic here into a CP.